packages/spl-utils/src/splToken.ts

```diff
@@ -14,7 +14,7 @@ export async function createMint(
   if (authority === undefined) {
     authority = provider.wallet.publicKey;
   }
-  const mint = new Account();
+  const mint = Keypair.generate();
```

Comment:
What happened here?

Reply:
Account is deprecated and it was an easy change to switch it out to Keypair in this case